.deployment.config.json

```diff
@@ -9,9 +10,12 @@
       "sequential": ["dev", "qa", "prd"]
     },
     "team_jenkins": "searchuibuilds",
-    "start_environment_automatically": false,
+    "start_environment_automatically": true,
     "notifications": {
       "slack_channels": ["#searchuibuilds"]
+    },
+    "prd": {
+      "start_environment_automatically": false
     }
```

Comment:
This ensures the `prd` environment will only be executed when there's manual approval from QA.

Reply:
This is ok, but you could also do the opposite and have the default be false and set to true for dev and qa. That way, if you ever decide to add another environment (e.g.: hip), it won't start automatically.